packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import Observation from './common/observation/_Observation';
+import observationPropShape from './proptypes/observationPropTypes';
+import { checkIsStatusInError, ConditionalWrapper } from './common/utils';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: observationPropShape,
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const createIcons = (resultData) => (resultData.isUnverified
+  ? (<IconUnverified className={cx('icon-unverified')} />)
+  : (
+    <React.Fragment>
+      {resultData.isModified ? (<IconModified className={cx('icon-modified')} />) : null}
+      {resultData.hasComment ? (<IconComment className={cx('icon-comment')} />) : null}
+    </React.Fragment>
+  )
+);
+
+const createSecondaryDisplays = (resultData) => (
+  <React.Fragment>
+    {resultData.conceptDisplay ? (<div className={cx('concept-display')}>{resultData.conceptDisplay}</div>) : null}
+    {resultData.datetimeDisplay ? (<div className={cx('datetime-display')}>{resultData.datetimeDisplay}</div>) : null}
+  </React.Fragment>
+);
+
+const createClinicalResultDisplay = (resultData, hideUnit, isTruncated, hasResultError, hasResultNoData) => {
+  if (hasResultError) {
```

Comment:
Can we move this out of this helper? Seems like it should have been handled by `ClinicalResult`

Reply:
good suggestion, removed `hasResultError` and `hasResultNoData` from helper  - updated here: https://github.com/cerner/terra-clinical/pull/574/commits/f8046100436d9a56835661f232e9d56083866c0f